pkg/multiclustermanager/routes.go

```diff
@@ -93,11 +93,20 @@ func router(ctx context.Context, localClusterEnabled bool, tunnelAuthorizer *mcm
 	unauthed.PathPrefix("/v3-public").Handler(publicAPI)
 
 	// Authenticated routes
-	authed := mux.NewRouter()
-	authed.UseEncodedPath()
 	impersonatingAuth := auth.ToMiddleware(requests.NewImpersonatingAuth(sar.NewSubjectAccessReview(clusterManager)))
+	saAuth := auth.ToMiddleware(requests.NewServiceAccountAuth(scaledContext, clustermanager.ToRESTConfig))
 	accessControlHandler := rbac.NewAccessControlHandler()
 
+	saauthed := mux.NewRouter()
+	saauthed.UseEncodedPath()
+	saauthed.PathPrefix("/k8s/clusters/{clusterID}").Handler(k8sProxy)
```

Comment:
This new chain appears to be able to handle the same forms of auth as the standard path (unless I am missing something). Given that, do we still get value out of [the auth'd path prefix](https://github.com/rancher/rancher/blob/afc3811cb93466cf46fc25400064be0a5d958ef4/pkg/multiclustermanager/routes.go#L121) for this value?

Reply:
We think this is correct and we have removed the "auth'd path prefix" from the standard path.